src/injected/content/clipboard.js

```diff
@@ -23,18 +18,20 @@ bridge.addHandlers({
   },
 });
 
+// not using ::safe calls as it mysteriously doesn't work here in FF (probably a bug)
+
 function onCopy(e) {
-  e::stopImmediatePropagation();
-  e::preventDefault();
+  e.stopImmediatePropagation();
+  e.preventDefault();
   const { type, data } = clipboardData;
-  e::getClipboardData::setData(type || 'text/plain', data);
```

Comment:
It should be `e::getClipboardData()` instead of `e::getClipboardData` here.

Reply:
Gee, I **really** can be blind, but this still doesn't help in Firefox ESR: like I explained above FF fails in different places and ESR won't even run the onCopy listener with document::addEventListener, only with document.addEventListener

Reply:
Eek, FF ESR defines `execCommand` in `document.__proto__` not in `Document.prototype`...